src/litegraph.js

```diff
@@ -8239,70 +8262,75 @@ const globalExport = {};
                 return;
             }
 
-            this.graph.beforeChange();
+            this.emitBeforeChange();
```

Comment:
Can we rename `pasteFromClipboard` to `_pasteFromClipboard`, and add the wrapping logic there? I would like to avoid the large try block here.

Reply:
Done